FogCreek/Glitch-Community
# Pond powder: fix notification bug

## Links
* https://pond-powder.glitch.me
* https://glitch.manuscript.com/f/cases/3328730/

## Changes:
* fix notification id's so that they provide a random number. For reasons I don't totally understand this actually fixes a bug where the popover closes when the notification closes. It's weird because I can't figure out why the popover would close? But at least when I'm qa-ing it that's what I'm seeing

## How To Test:
* add a project to a collection, 
* see the notification at the top right hand side of the screen, 
* while the notification is still present,
* open the add project to collection popover again
* notice that the popover stays open when the notification auto-dismisses

## Feedback I'm looking for:
- do you see that this fixes the bug too?  
- do you understand why fixing this bug would help?
